x-pack/plugins/serverless_search/public/application/components/languages/php.ts

```diff
@@ -35,111 +29,39 @@ print_r($response->asArray());`,
-'release_date' => '1992-06-01',
-'page_count' => 470,
-],
-[
-'index' => [
-'_index' => 'books',
-'_id' => '9780441017225',
-],
-],
-[
-'name' => 'Revelation Space',
-'author' => 'Alastair Reynolds',
-'release_date' => '2000-03-15',
-'page_count' => 585,
-],
-[
-'index' => [
-'_index' => 'books',
-'_id' => '9780451524935',
-],
-],
-[
-'name' => '1984',
-'author' => 'George Orwell',
-'release_date' => '1985-06-01',
-'page_count' => 328,
-],
-[
-'index' => [
-'_index' => 'books',
-'_id' => '9781451673319',
-],
-],
-[
-'name' => 'Fahrenheit 451',
-'author' => 'Ray Bradbury',
-'release_date' => '1953-10-15',
-'page_count' => 227,
-],
-[
-'index' => [
-'_index' => 'books',
-'_id' => '9780060850524',
-],
-],
-[
-'name' => 'Brave New World',
-'author' => 'Aldous Huxley',
-'release_date' => '1932-06-01',
-'page_count' => 268,
-],
-[
-'index' => [
-'_index' => 'books',
-'_id' => '9780385490818',
-],
-],
-[
-'name' => 'The Handmaid\'s Tale',
-'author' => 'Margaret Atwood',
-'release_date' => '1985-06-01',
-'page_count' => 311,
-],
-],
+  ingestData: `$body = [
+    [ "index" => [ "_index" => "books" ]],
+    [ "name" => "Snow Crash", "author" => "Neal Stephenson", "release_date" => "1992-06-01", "page_count" => 470],
+    [ "index" => [ "_index" => "books" ]],
+    [ "name" => "Revelation Space", "author" => "Alastair Reynolds", "release_date" => "2000-03-15", "page_count" => 585],
+    [ "index" => [ "_index" => "books" ]],
+    [ "name" => "1984", "author" => "George Orwell", "release_date" => "1949-06-08", "page_count" => 328],
+    [ "index" => [ "_index" => "books" ]],
+    [ "name" => "Fahrenheit 451", "author" => "Ray Bradbury", "release_date" => "1953-10-15", "page_count" => 227],
+    [ "index" => [ "_index" => "books" ]],
+    [ "name" => "Brave New World", "author" => "Aldous Huxley", "release_date" => "1932-06-01", "page_count" => 268],
+    [ "index" => [ "_index" => "books" ]],
+    [ "name" => "The Handmaid's Tale", "author" => "Margaret Atwood", "release_date" => "1985-06-01", "page_count" => 311]
 ];
 
-$response = $client->bulk($params);
+$response = $client->bulk(body: $body);
 echo $response->getStatusCode();
 echo (string) $response->getBody();`,
   ingestDataIndex: ({ apiKey, url, indexName }) => `$client = ClientBuilder::create()
-  ->setHosts(['${url}'])
+  ->setEndpoint('${url}')
   ->setApiKey('${apiKey}')
   ->build();
 
-$params = [
-'body' => [
-[
-'index' => [
-'_index' => '${indexName ?? INDEX_NAME_PLACEHOLDER}',
-'_id' => '1',
-],
-],
-[
-'name' => 'foo',
-'title' => 'bar',
-],
-],
+$body = [
+  [ 'index' => [ '_index' => '${indexName ?? INDEX_NAME_PLACEHOLDER}', '_id' => '1' ]],
+  [ 'name' => 'foo', 'title' => 'bar' ]
 ];
 
-$response = $client->bulk($params);
+$response = $client->bulk(body: $body);
 echo $response->getStatusCode();
 echo (string) $response->getBody();
 `,
-  installClient: 'composer require elasticsearch/elasticsearch',
+  installClient: 'composer require elastic/elasticsearch-serverless:0.1.0-alpha1',
```

Comment:
question: Does it make sense to refer to a specific alpha version here, or would `elastic/elasticsearch-serverless` be sufficient?

Reply:
Good point. Right now we are going to release only alpha/beta releases and people need to specify it when installing. In the future, when we'll have the first 1.0.0 stable we will remove it having only `elastic/elasticsearch-serverless`. I think we should offer it at the moment, otherwise people will not be able to install the PHP library.

Reply:
Makes sense, I'll create an issue so we can track that we should replace this when we first release a stable client.